app/models/course/assessment/category.rb

```diff
@@ -40,7 +42,7 @@ def build_initial_tab
 
   def build_initial_folder
     build_folder(name: title, course: course, parent: course.root_folder,
-                 start_at: Time.zone.now)
+                 start_at: Time.zone.now) unless folder
```

Comment:
then this should be split from the build_initial_folder with the assign_folder_attributes